crates/solana-event-forwarder/src/component.rs

```diff
@@ -238,23 +413,122 @@ fn map_gateway_event_to_amplifier_event(
                             .build(),
                     )
                     .build(),
-            );
+            ));
             tracing::info!("message approved");
-            Some(amplifier_event)
         }
-        GatewayEvent::MessageExecuted(ref _executed_message) => {
+        GatewayAndGasEvent::MessageExecuted(ref _executed_message) => {
             tracing::warn!(
                 "current gateway event does not produce enough artifacts to relay this message"
             );
-            None
         }
-        GatewayEvent::OperatorshipTransferred(ref new_operatorship) => {
-            tracing::info!(?new_operatorship, "Operatorship transferred");
-            None
+        GatewayAndGasEvent::NativeGasRefunded(event) => {
```

Comment:
Can't we use `construct_gas_event` for these last two patterns as well?

Reply:
I updated the code to use the helper in `NativeGasAdded`, but `NativeGasRefunded` requires a different type, so that one will say as is